src/com/lilithsthrone/game/character/effects/StatusEffect.java

```diff
@@ -5462,7 +5462,7 @@ public String extraRemovalEffects(GameCharacter target) {
 		@Override
 		public boolean isConditionsMet(GameCharacter target) {
 			float cumAmount = target.getTotalFluidInArea(SexAreaOrifice.NIPPLE);
-			return cumAmount >= CumProduction.SEVEN_MONSTROUS.getMaximumValue()
+			return cumAmount >= Math.pow(((Math.cbrt(30)/182.88)*target.getHeightValue())*((target.getBreastSize().getMeasurement()/3)+1),3)
```

Comment:
This calculation is almost equal to `Math.pow(target.getHeightValue()*((target.getBreastSize().getMeasurement()/3)+1), 3) / 203881` (~-0.00014% difference) and is a much simpler calculation.